### PR TITLE
fix: do not add ROUTER_PROVIDERS on generation.

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

File: addon/ng2/blueprints/ng2/files/src/client/main.ts [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+import {ROUTER_PROVIDERS} from 'angular2/router'
 import {environment} from './app/environment';
 import {<%= jsComponentName %>App} from './app/<%= htmlComponentName %>.component';
 
 if (environment.production) {
   enableProdMode();
 }
 
-bootstrap(<%= jsComponentName %>App);
+bootstrap(<%= jsComponentName %>App, [
```
Comment (johnpapa) [inline addon/ng2/blueprints/ng2/files/src/client/main.ts]: why add them here? the style guide advice (and from Igor) has been to not add providers in the main bootstrap, but instead to put them in the root app component

Comment (hansl) [inline addon/ng2/blueprints/ng2/files/src/client/main.ts]: I'm okay with that too

Comment (johnpapa) [inline addon/ng2/blueprints/ng2/files/src/client/main.ts]: OK. That would be great to move this to the root app component, so we stay in sync

Comment (hansl) [inline addon/ng2/blueprints/ng2/files/src/client/main.ts]: done.

File: addon/ng2/blueprints/ng2/files/src/client/app/__name__.component.ts [hunk 2]
```diff
@@ -1,17 +1,13 @@
 import {Component} from 'angular2/core';
-import {RouteConfig, ROUTER_DIRECTIVES, ROUTER_PROVIDERS} from 'angular2/router';
```
Comment (filipesilva) [inline addon/ng2/blueprints/ng2/files/src/client/app/__name__.component.ts]: This component still has a `router-outlet`, which means it needs `ROUTER_DIRECTIVES` I believe.

Comment (hansl) [inline addon/ng2/blueprints/ng2/files/src/client/app/__name__.component.ts]: done.

**Comment by hansl** [comment]
@filipesilva: Please take another look. It passes the tests now.

**Pull request merged**